doc/source/user/quickstart.rst

```diff
@@ -79,8 +79,8 @@ An example
     >>> a = np.arange(15).reshape(3, 5)
     >>> a
     array([[ 0,  1,  2,  3,  4],
-           [ 5,  6,  7,  8,  9],
-           [10, 11, 12, 13, 14]])
+	   [ 5,  6,  7,  8,  9],
+	   [10, 11, 12, 13, 14]])
```

Comment:
This change looks wrong to me

Reply:
I did a lot of copy-paste, tabs got mixed in by mistake. Funnily, it passes pytest and refguide_check. Fix coming.